styles.js

```diff
@@ -67,6 +67,7 @@ export default createGlobalStyle`
     background-color: var(--accentColor);
     width: 100%;
     box-shadow: 0px 1px 2px var(--primaryDarkColor);
+    z-index: 1;
```

Comment:
same as above: the MapContainer is not part of the `<main>` but an independent element of the DOM. So for now, the z-index is our solution, to prevent unwanted overlaps with the `<header>` or other `<main>` - components...